sdk/ai/azure-ai-agents/azure/ai/agents/models/_models.py

```diff
@@ -883,6 +884,34 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         super().__init__(*args, **kwargs)
 
 
+class BingGroundingConnectionInformation(_Model):
```

Comment:
What exactly is the purpose of this new class, is it intended to replace an existing class? I'm not seeing where it's actually used in existing code.

Reply:
@glharper the class `BingGroundingConnectionInformation` is used by the constructor of the hand-written class `DeepResearchTool` (new in this PR). As far as I can tell, we can make this class internal. I'll following on this. Also see my comment in the T[ypeSpec PR,](BingGroundingConnectionInformation) as I think it should be renamed to `DeepResearchBingGroundingConnection`, to indicate that this is used for Deep Research only and differentiate it from the other `BingGrounding` classes.